build.sbt

```diff
@@ -102,10 +103,14 @@ lazy val root = (project in file(".")).
     buildInfoPackage := "com.databricks.labs.deltaoms.common",
     assembly / test  := {},
     assembly / assemblyOption := (assembly / assemblyOption).value.copy(includeScala = false),
-    assembly / assemblyShadeRules := Seq(
-      ShadeRule.rename("org.apache.spark.sql.delta.**" ->
-        "com.databricks.sql.transaction.tahoe.@1").inAll
-    ),
+    if (buildTarget == "db") {
```

Comment:
Make it only-Databricks target 😉 it must not run outside of Databricks.

Reply:
@nfx , ok. Made the changes